src/SelfGravity/CPU_Poisson/CPU_ComputeGREP.cpp

```diff
@@ -151,37 +172,40 @@ void CPU_ComputeGREP( const int lv, const double Time, const Profile_t *DensAve,
    } // if ( ! IsConverged )
 
 
-// 3. compute the effective GR potential at the bin center
+// (3) compute the effective GR potential, defined at the left edge of each bin
    EffPot->NBin        = DensAve->NBin;
    EffPot->LogBin      = DensAve->LogBin;
    EffPot->LogBinRatio = DensAve->LogBinRatio;
    EffPot->MaxRadius   = DensAve->MaxRadius;
    EffPot->AllocateMemory();
 
    for (int d=0; d<3;    d++)   EffPot->Center[d] = DensAve->Center[d];
-   for (int b=0; b<NBin; b++)   EffPot->Radius[b] = DensAve->Radius[b];
+   for (int b=0; b<NBin; b++)   EffPot->Radius[b] = Edge_L[b];
 
-// set the outer boundary condition of the potential to -G * M_out / r_out
-   EffPot->Data[NBin-1] = -NEWTON_G * ( Mass_TOV[NBin-1] - Mass_NW[NBin-1] ) / Radius[NBin-1];
+// set the outer boundary condition of the potential to -GM/r at the left edge of last bin
```

Comment:
Clarify why we subtract the Newtonian potential here.

Reply:
Comments updated.